systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/JobUtils.java

```diff
@@ -20,6 +22,17 @@
 
     private JobUtils() { }
 
+    /**
+     * Wait until all Jobs are deleted in given namespace.
+     * @param namespace Delete all jobs in this namespace
+     */
+    public static void removeAllJobs(String namespace) {
+        JobList jobs = kubeClient().namespace(namespace).getJobList();
+        for (Job job : jobs.getItems()) {
+            deleteJobWithWait(namespace, job.getMetadata().getName());
+        }
```

Comment:
```suggestion
        kubeClient().namespace(namespace).getJobList().getItems().stream().forEach(job -> JobUtils.deleteJobWithWait(namespace, job.getMetadata().getName()));
```
maybe something like this?